exasol/python_extension_common/deployment/language_container_validator.py

```diff
@@ -0,0 +1,137 @@
+from __future__ import annotations
+from typing import Generator
+from datetime import timedelta
+import random
+import string
+from contextlib import contextmanager
+from textwrap import dedent
+
+from tenacity import retry
+from tenacity.wait import wait_fixed
+from tenacity.stop import stop_after_delay, stop_after_attempt
+
+import pyexasol     # type: ignore
+
+_DUMMY_UDF_NAME = 'DUMMY_UDF'
+
+
+def _dress_schema(schema: str | None) -> str:
+
+    if schema:
+        return f'"{schema}".'
+    return ''
+
+
+def _create_dummy_udf(conn: pyexasol.ExaConnection, language_alias: str,
+                      schema: str | None) -> None:
+
+    sql = dedent(f"""
+    CREATE OR REPLACE {language_alias} SCALAR SCRIPT {_dress_schema(schema)}"{_DUMMY_UDF_NAME}"()
+    RETURNS DECIMAL(1, 0) AS
+
+    def run(ctx):
+        return 0
+    /
+    """)
```

Comment:
Sometimes, if the text and/or template is bigger or more complex, evaluations for parts of the template are required. I find that the old style `string-format` can improve the readability of the template and evaluations.

```suggestion
    sql = dedent(
    """
    CREATE OR REPLACE {lang} SCALAR SCRIPT {schema}"{udf_name}"()
    RETURNS DECIMAL(1, 0) AS

    def run(ctx):
        return 0
    /
    """.format(
        lang=language_alias,
        schema=_dress_schema(schema),
        udf_name=_DUMMY_UDF_NAME
    ))

```

Reply:
Not sure. I think if I work out the full name (as per the next comment) in a separate line, this will make the code most compact and readable.